src/core/internal/traits.d

```diff
@@ -274,6 +274,23 @@ template hasElaborateDestructor(S)
     }
 }
 
+// std.traits.hasElaboratePostblit
```

Comment:
I don't see `hasElaboratePostblit` in std.traits, what does this mean? 

A full DDoc comment would be appreciated.